src/hotspot/cpu/aarch64/aarch64_sve_ad.m4

```diff
@@ -971,40 +985,36 @@ instruct vstoremaskL(vReg dst, vReg src, vReg tmp, immI_8 size) %{
 
 // load/store mask vector
 
-instruct vloadmask_loadV(vReg dst, vmemA mem) %{
-  predicate(UseSVE > 0 && n->as_Vector()->length_in_bytes() >= 16 &&
-            n->in(1)->bottom_type()->is_vect()->element_basic_type() == T_BOOLEAN);
-  match(Set dst (VectorLoadMask (LoadVector mem)));
-  ins_cost(2 * SVE_COST);
-  format %{ "sve_ld1b $dst, $mem\n\t"
-            "sve_neg $dst, $dst\t # load vector mask (sve)" %}
+instruct loadVMask(vReg dst, vmemA mem) %{
+  predicate(UseSVE > 0);
+  match(Set dst (LoadVectorMask mem));
```

Comment:
Are we trying to perform a vector load operation under the influence of a mask here? 
C2 already has an ideal node LoadVectorMaskedNode which may come handy.